libbeat/docs/upgrading.asciidoc

```diff
@@ -1,160 +1,324 @@
 
-See <<breaking-changes-import-dashboards>> in the Breaking Changes
-documentation.
+Staring with {beats} 7.0, index lifecycle management is on by default when
+sending data to {beats} clusters that support it. Make sure {beats} users have
+the privileges needed to use index lifecycle management, or disable index
+lifecycle management. The following privileges are required:
 
-NOTE: If you've modified the Beats dashboards manually, reimporting them will
-overwrite your changes. Consider saving your dashboards under new IDs, so
-that the reimport doesn't overwrite them.
+* To set up index lifecycle policies: `manage_ilm` and `monitor` on cluster
+* To send data to a secured cluster when index lifecycle management is enabled:
+`manage_index_templates` and `monitor` on cluster, and `index` and `manage` on
+{beats} indices.
 
-==== Heartbeat package names
+// REVIEWERS: The privileges shown here are based on testing I did for
+// https://github.com/elastic/beats/pull/11329, but I am still waiting for
+// more review comments on that PR. 
 
-The DEB and RPM packages for Heartbeat are now named `heartbeat-elastic` to
-avoid conflicts with a different `heartbeat` project.  The `.deb` and `.rpm`
-file names haven't changed, just the package name in the repositories.
+To disable index lifecycle management, set `setup.ilm.enabled: false` in the
+{beats} configuration file.
 
 [[upgrade-mapping-template]]
-=== Upgrade the Elasticsearch mapping template
+=== Upgrade the {es} mapping template
 
-Starting with Beats 6.0, the mapping templates and the default index names are
-versioned. For example, Metricbeat 6.0.0 creates indices like this:
+// REVIEWERS: Does this topic really belong on its own rather than as a
+// sub-topic? Do we want to tell users to set 
 
-[source,shell]
+Mapping templates and the default index names are versioned. For example,
+Metricbeat {stack-version} typically creates indices like this:
+
+["source","sh",subs="attributes"]
 ------------------------------------------------------------------------------
-metricbeat-6.0.0-2017-08-31
+metricbeat-{stack-version}-{localdate}
 ------------------------------------------------------------------------------
 
-And the corresponding Elasticsearch template is named `metricbeat-6.0.0`.
+And the corresponding {es} template is named +metricbeat-{stack-version}+.
 
 This means that each version of the Beat creates a new index, and it is
-guaranteed that the correct template for that version is applied.  With these
+guaranteed that the correct template for that version is applied. With these
 changes in place, you generally don't have to do anything to upgrade the mapping
 template when you move to a new version.
 
-One exception is when upgrading from 5.x, in which case you should make sure to
-read the <<upgrading-to-5.6>> section.
+// REVIEWERS: I'm not sure what the following paragraph means. Why would this
+// result in downtime?
```

Comment:
I guess it means that downtime will occur if the index/template is not versioned. Beats 5.x (see original text) did not version those I think.

Reply:
Removing because it sounds like the statement is no longer relevant.